backend/app/models/user.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  has_one :surveyor
```

Comment:
Had to add this relationship so that `surveyor` would be available below but the linter is complaining this needs `:dependent` and I'm not sure what the appropriate value for that would be.

Reply:
TODO: add `dependent: :destroy`